tests/gtests/test_SW_Weather.cc

```diff
@@ -25,10 +25,10 @@
 
 namespace {
 
-    TEST(ReadAllWeatherTest, DefaultValues) {
+    TEST_F(AllTest, DefaultValues) {
```

Comment:
Please improve test suit names here and throughout following examples from my earlier comments